src/dal/Jalasoft.TeamUp.Projects.DAL.Interfaces/IRepository.cs

```diff
@@ -11,5 +11,7 @@ public interface IRepository<T>
         public IEnumerable<T> GetAll();
 
         public T Add(T project);
+
+        public T DeleteById(Guid id);
```

Comment:
please change the name to RemoveProject to follow Repository pattern conventions